js/scenes/SignUpScreen/SignUpScreen.js

```diff
@@ -49,10 +46,9 @@ class SignupForm extends Gandalf {
       },
       {
         name: 'password',
-        component: TextInput,
+        component: ValidatedText,
         validators: ['required'],
         errorPropName: 'error',
-        onChangeHandler: 'onChangeText',
         props: {
           placeholder: 'Password',
           style: styles.login
```

Comment:
You're going to want to add the `secureTextEntry: true` prop here. This will make it behave like a password input.